packages/eddsa-poseidon/tests/index.test.ts

```diff
@@ -385,8 +385,8 @@ describe("EdDSAPoseidon", () => {
 
```

Comment:
What specific bugs or use cases are you trying to improve here?  Can you create a specific test case which demonstrates an issue so you can then demonstrate that it's fixed?